### PR TITLE
Update functionMap.php

### PR DESCRIPTION
**Pull request by colinmo**
Added option of SQLite3Stmt::execute to return false (bool) if it failed.

Issue Ref: https://github.com/phpstan/phpstan/issues/3605
PHPDoc Ref: https://www.php.net/manual/en/sqlite3stmt.execute.php

### CODE REVIEW

File: src/Reflection/SignatureMap/functionMap.php [hunk 1]
```diff
@@ -11442,7 +11442,7 @@
 'SQLite3Stmt::bindValue' => ['bool', 'parameter_name_or_number'=>'string|int', 'parameter'=>'mixed', 'type='=>'int'],
 'SQLite3Stmt::clear' => ['bool'],
 'SQLite3Stmt::close' => ['bool'],
-'SQLite3Stmt::execute' => ['SQLite3Result'],
+'SQLite3Stmt::execute' => ['bool|SQLite3Result'],
```
Comment (ondrejmirtes) [inline src/Reflection/SignatureMap/functionMap.php:11445]: should be just false

File: tests/PHPStan/Rules/Methods/data/returnTypes.php [hunk 2]
```diff
@@ -349,7 +349,7 @@ public function returnsNullInTernary(): int
 		}
 	}
 
-	public function misleadingBoolReturnType(): boolean
+	public function misleadingBoolReturnType(): bool
```
Comment (ondrejmirtes) [inline tests/PHPStan/Rules/Methods/data/returnTypes.php:352]: I don't want these changes, they're here on purpose

File: src/Reflection/SignatureMap/functionMap.php (hunk 1 above)
Comment (colinmo) [reply src/Reflection/SignatureMap/functionMap.php]: Repaired, thank you.

File: tests/PHPStan/Rules/Methods/data/returnTypes.php (hunk 2 above)
Comment (colinmo) [reply tests/PHPStan/Rules/Methods/data/returnTypes.php]: I apologise, restored, thank you.

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**